src/login/LoginViewController.js

```diff
@@ -228,6 +229,29 @@ export class LoginViewController implements ILoginViewController {
 				import("../native/common/FileApp")
 					.then(({fileApp}) => fileApp.clearFileData())
 					.catch((e) => console.log("Failed to clean file data", e))
+
+				return logins.getUserController().loadCustomerInfo().then(customerInfo => {
+					// Get a whitelabelConfig if it exists
+					const whiteLabelConfigId = customerInfo.domainInfos
+					                                       .map(domainInfo => domainInfo.whitelabelConfig)
+					                                       .filter(Boolean)[0]
+
+					if (!!whiteLabelConfigId) {
```

Comment:
Do we need to do this for then web client? It should already be in whitelabelCustomizations and we can avoid the request